src/rovr_control/rovr_control/main_control_node.py

```diff
@@ -37,19 +37,20 @@
 states = {"Teleop": 0, "Auto_Dig": 1, "Auto_Offload": 2}
 
 
-def get_target_ip(target: str, default: str = "", logger_fn=print) -> str:
```

Comment:
Just delete this, it exists in old version of code if needed. No need for it to be commented